src/components/campaigns/Summary/GoalProgress/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+const GoalProgress = ({ goal = 0, fundsRaisedAmount = 0 }) => {
+  if (!goal) return <span />
+  const progressPercentage = Math.min(fundsRaisedAmount / goal, 1) * 100
+  return (
+    <div key='progressbar' className='CampaignSummary__progressbar'>
```

Comment:
I feel this class name should be `CampaignSummaryProgressBar` and the children should follow suit to match BEM.